src/RichTextEditor.js

```diff
@@ -45,6 +46,7 @@ type Props = {
   handleReturn?: (event: Object) => boolean;
   readOnly?: boolean;
   disabled?: boolean; // Alias of readOnly
+  toolbarConfig?: EditorToolbarConfig;
```

Comment:
I believe this line is incorrect. Th right hand side must be a type annotation, not an object (although the two looks similar at times). I'll try to come up with an example of how to fix it.